src/Game/Renderer.java

```diff
@@ -574,6 +574,15 @@ public static void present(Graphics g, Image image) {
           x2 -= barSize;
         }
       }
+      
+      // Draw last menu action
+      if (!Client.isInterfaceOpen() && !Client.show_questionmenu 
+    		  && Settings.SHOW_LAST_MENU_ACTION.get(Settings.currentProfile)) {
+    	  if (time <= Client.menu_timer) {
+              drawShadowText(g2, Client.lastAction, x, y, color_text, false);
```

Comment:
Don't really like this. It would be cooler to highlight the tile interacted with, or at least to output to console @ "Info" level (only if "SHOW_LAST_MENU_ACTION" is true).

Right now it's very easy to miss.

Reply:
Can throw in also the show to console, any desired pattern to have it displayed? like (MENU_ACTION) [TALK_TO] ?

Reply:
Probably just `(ACTION)` because it will fit the 6 character limit & normally players wouldn't think of casting magic or clicking on an anvil as a "menu".

Reply:
Placed but since left it as Logger.Info in the main point, think just has to be one or another if its Logger.Info, or have Client.displayMessage, passing it as type ACTION, which one will be preferred?

Reply:
Logger.Info seems fine actually.